pkg/target/conduit/configuration.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright (c) 2021 Nordix Foundation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conduit
+
+import (
+	"context"
+	"io"
+
+	nspAPI "github.com/nordix/meridio/api/nsp/v1"
+	"github.com/nordix/meridio/pkg/target/types"
+	"github.com/sirupsen/logrus"
+)
+
+type Configuration struct {
+	Conduit                    types.Conduit
+	ConfigurationManagerClient nspAPI.ConfigurationManagerClient
+	Cancel                     context.CancelFunc
+}
+
+type SetVips interface {
```

Comment:
Is this used at all?

Reply:
No, I had the plan to use it to replace the "Conduit" in the Configuration struct, so it would be easier to create unit tests later with a small mock of it.